webapp/utils/utils.jsx

```diff
@@ -314,14 +313,9 @@ export function getTimestamp() {
 }
 
 // extracts links not styled by Markdown
-export function extractLinks(text) {
-    text; // eslint-disable-line no-unused-expressions
-    Autolinker; // eslint-disable-line no-unused-expressions
-
-    // skip this operation because autolinker is having issues
-    return [];
-
-    /*const links = [];
+export function extractFirstLink(text) {
+    const pattern = /(^|[\s\n]|<br\/?>)((?:https?|ftp):\/\/[\-A-Z0-9+\u0026\u2019@#\/%?=()~_|!:,.;]*[\-A-Z0-9+\u0026@#\/%=~()_|])/gi;
```

Comment:
@hmhealey removing the /g from the regex causes this:

![image](https://cloud.githubusercontent.com/assets/6757047/14213583/25fee002-f80f-11e5-851d-2ea137d8108f.png)

with the `/g` the result is `[" http://google.com", " http://github.com"]` so I would prefer to keep it